### ISSUE TITLE
Failed to load the rainbow-delimiters package

### ISSUE THREAD

**Issue by rreinhardt9** [paradoxxxzero/atom-rainbow-delimiters#1]
[Enter steps to reproduce:]

1. Install package
2. Error will be shown after install and each subsequent editor restart.

Seems like themes have to implement a `syntax-variables.less`. My thought is that this might not be a typical variable that themes have to implement? None of mine have it at least, but I really have no idea which ones are typical/ required :-)

**Atom**: 1.14.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.11.6
**Thrown From**: [rainbow-delimiters](https://github.com/paradoxxxzero/atom-rainbow-delimiters) package 1.2.0

### Stack Trace

Failed to load the rainbow-delimiters package

```
At variable @syntax-accent is undefined in /Users/ross/.atom/packages/rainbow-delimiters/styles/rainbow-delimiters.less:19:0

LessError: variable @syntax-accent is undefined
  at /packages/rainbow-delimiters/styles/rainbow-delimiters.less:19:0
```

### Commands

```
```

### Non-Core Packages

```
blame 0.10.2 
browser-refresh 0.9.0 
dash 1.7.0 
editorconfig 2.2.0 
fonts 2.0.0 
language-elixir 0.16.0 
language-haml 0.24.1 
language-r 0.4.1 
linter 1.11.21 
linter-coffeelint 1.2.1 
linter-eslint 8.1.0 
linter-rubocop 0.5.3 
markdown-toc 0.4.2 
material 2.2.0 
open-in-browser 0.4.7 
rails-snippets 2.11.0 
rails-transporter 1.6.5 
railscast-theme 0.3.0 
rainbow-delimiters 1.2.0 
react 0.16.2 
sort-lines 0.14.0 
tasks 2.6.6 
vim-mode 0.66.0 
```

**Comment by paradoxxxzero** [comment]
I should be better :)
Thanks for reporting.

**Comment by rreinhardt9** [comment]
I just got the fix in 1.3.0, and it's working great! I'm really loving the package... thanks!! ❤️
